website/blog/2021-03-11-jest-27.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
Gotta remember to update date

Reply:
tuesday 25th is the plan if you wanna prep it

Reply:
I put that for now :+1: